coordinators/build.gradle

```diff
@@ -17,6 +17,7 @@ android {
 
 dependencies {
   compile 'com.android.support:support-annotations:23.3.0'
+  compile 'com.android.support:support-v4:23.3.0'
```

Comment:
Talking about inlining the implementation from ViewCompat. I'll take a look at it and maybe post a followup.